testData/results/TestIdeaNotNull.dec

```diff
@@ -9,26 +9,58 @@ public class TestIdeaNotNull {
       String s = "";// 10
       int i = new Random().nextInt(12);// 11
       if (i == 0) {// 13
-         return s;// 14
+         if (s == null) {// 14
+            $$$reportNull$$$0(0);
+         }
+
+         return s;
```

Comment:
does it make sense for these tests to still be here?

Reply:
Yep, there's a comment saying that this is here to show the default behavior